install/update_91_92.php

```diff
@@ -1294,6 +1294,26 @@ function update91to92() {
       $migration->dropTable('glpi_networkequipmentfirmwares');
    }
 
+   //add editor in followupps
+   if (!fieldExists('glpi_ticketfollowups', 'editors_id')) {
```

Comment:
This should be called "fieldExists"... I've begin to work on a PR to fix method names; this is a huge job :D I'lle change to use current method name anyways.